src/core/api/index.js

```diff
@@ -8,9 +9,14 @@ import { schema as normalizrSchema, normalize } from 'normalizr';
 import { oneLine } from 'common-tags';
 import config from 'config';
 
+import { initialApiState } from 'core/reducers/api';
 import { ADDON_TYPE_THEME } from 'core/constants';
 import log from 'core/logger';
 import { convertFiltersToQueryParams } from 'core/searchUtils';
+/* eslint-disable no-duplicate-imports */
```

Comment:
Nope:
````
src/core/api/index.js:12
 12: import { initialApiState }, type { ApiStateType } from 'core/reducers/api';
                               ^ Unexpected token ,
````
There are hundreds of issues open about "import type" so I'm not sure if someone is thinking about this or not.